From 0e85fc8872a0b865b22a11bc16676bd5e22be7c0 Mon Sep 17 00:00:00 2001
From: Scott Fang <zhouzhou.fang@gmail.com>
Date: Thu, 21 Feb 2013 13:24:54 +0800
Subject: [PORT FROM MAIN] (2/2)Enable triple frame buffer support.

BZ:90509

HDMI surface is based on SF composition, sometimes
SF can't get HDMI surface in time cause lag, although
App itself has finish render its own buffer.

Change-Id: I368dd30d63569fef09189b01da176ae4997dfd85
Orig-Change-Id: I12211a22b009e6922ce9a78c38b8e12708ce33cb
Thanks to Marc Mao's support
Signed-off-by: Scott Fang <zhouzhou.fang@gmail.com>
Reviewed-on: http://android.intel.com:8080/94762
Tested-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: Tong, BoX <box.tong@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 .../DisplayHardware/FramebufferSurface.cpp         |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp b/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp
index 54a3ce8..d02d07a 100644
--- a/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp
+++ b/services/surfaceflinger/DisplayHardware/FramebufferSurface.cpp
@@ -37,7 +37,7 @@
 #include "HWComposer.h"
 
 #ifndef NUM_FRAMEBUFFER_SURFACE_BUFFERS
-#define NUM_FRAMEBUFFER_SURFACE_BUFFERS (2)
+#define NUM_FRAMEBUFFER_SURFACE_BUFFERS (3)
 #endif
 
 // ----------------------------------------------------------------------------
-- 
1.7.4.1

